source/user-manual/api/rbac/reference.rst

```diff
@@ -148,6 +151,7 @@ This reference also contains a set of default roles and policies that can be imm
     - `task_status`_
     - `users_all`_
     - `vulnerability_read`_
+    - `events_ingest`_
```

Comment:
Same as the previous one, it could be placed after `decoders_read` and before `lists_all` to maintain the alphabetical order.